scripts/gen_binding.py

```diff
@@ -418,7 +418,10 @@ def normalize_arg(arg):
           c_impl.append(f"g_bytes_used += t.bytes();")
           c_impl.append(f"return new fl::Tensor(t);")
           c_impl.append("} else {")
-        c_impl.append(f"auto t = fl::{op}({c_args});")
+        if op == 'where':
```

Comment:
I'd recommend doing this at like 326 of this file (c_op_args, before they're merged)

Reply:
Fixed!